config/fund_loader_config/cyp/cyp_r1.py

```diff
@@ -0,0 +1,165 @@
+from datetime import datetime
+from datetime import timezone
+
+from config.fund_loader_config.common_fund_config.fund_base_tree_paths import (
+    CYP_R1_BASE_PATH,
+)
+from config.fund_loader_config.logo import DLUHC_LOGO_PNG
+
+CYP_FUND_ID = "1baa0f68-4e0a-4b02-9dfe-b5646f089e65"
+CYP_ROUND_1_ID = "888aae3d-7e2c-4523-b9c1-95952b3d1644"
+APPLICATION_BASE_PATH = ".".join([str(CYP_R1_BASE_PATH), str(1)])
+ASSESSMENT_BASE_PATH = ".".join([str(CYP_R1_BASE_PATH), str(2)])
+CYP_R1_OPENS_DATE = datetime(
+    2023, 9, 22, 12, 0, 0, tzinfo=timezone.utc
+)  # 2023-09-22 12:00:00
+CYP_R1_DEADLINE_DATE = datetime(
+    2023, 10, 31, 11, 59, 0, tzinfo=timezone.utc
+)  # 2023-10-31 11:59:00
+CYP_R1_ASSESSMENT_DEADLINE_DATE = datetime(
+    2023, 12, 24, 12, 0, 0, tzinfo=timezone.utc
+)  # 2023-12-24 12:00:00
+
+CYP_PROSPECTS_LINK = "https://www.gov.uk/government/publications/cyp-round-1-prospectus"
```

Comment:
This isn't a valid link - it 404s, is this a placeholder or will it show eventually?

Reply:
empty string for now, will add in a seperate PR when we have this available 👍🏾